[Bug]: ci failed for manifest opensearch-2.1.0.yml

### Describe the bug

`  ./ci.sh manifests/2.1.0/opensearch-2.1.0.yml --snapshot` failed with this error `Project 'plugin' not found in root project 'opensearch-sql'.`

The complete error is 

```
2022-07-04 03:18:01 INFO     Executing "./gradlew plugin:dependencies -Dopensearch.version=2.1.0-SNAPSHOT -Dbuild.snapshot=true --configuration compileOnly | grep -e "---"" in /tmp/tmpn5q5akha/sql
FAILURE: Build failed with an exception.
* What went wrong:
Project 'plugin' not found in root project 'opensearch-sql'.
* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
* Get more help at https://help.gradle.org/
BUILD FAILED in 1s
2022-07-04 03:18:06 ERROR    CI Manifest check failed
Traceback (most recent call last):
  File "./src/run_ci.py", line 25, in <module>
    sys.exit(main())
  File "./src/run_ci.py", line 20, in main
    CiManifests.from_file(args.manifest, args).check()
  File "/home/runner/work/opensearch-build/opensearch-build/src/ci_workflow/ci_manifest.py", line 21, in check
    self.__check__()
  File "/home/runner/work/opensearch-build/opensearch-build/src/ci_workflow/ci_input_manifest.py", line 41, in __check__
    ci_check_list.check()
  File "/home/runner/work/opensearch-build/opensearch-build/src/ci_workflow/ci_check_list_source.py", line 43, in check
    instance = klass(self.component, self.git_repo, self.target, check.args)
  File "/home/runner/work/opensearch-build/opensearch-build/src/ci_workflow/ci_check_gradle_dependencies.py", line 22, in __init__
    self.dependencies = self.__get_dependencies()
  File "/home/runner/work/opensearch-build/opensearch-build/src/ci_workflow/ci_check_gradle_dependencies.py", line 39, in __get_dependencies
    lines = self.git_repo.output(cmd)
  File "/home/runner/work/opensearch-build/opensearch-build/src/git/git_repository.py", line 78, in output
    return subprocess.check_output(command, cwd=cwd, shell=True).decode().strip()
  File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/subprocess.py", line 411, in check_output
    **kwargs).stdout
  File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/subprocess.py", line 512, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command './gradlew plugin:dependencies -Dopensearch.version=2.1.0-SNAPSHOT -Dbuild.snapshot=true --configuration compileOnly | grep -e "---"' returned non-zero exit status 1.
Error: Process completed with exit code 1.
```

See this run for details https://github.com/opensearch-project/opensearch-build/runs/7173513928?check_suite_focus=true


### To reproduce

Run `  ./ci.sh manifests/2.1.0/opensearch-2.1.0.yml --snapshot`

### Expected behavior

_No response_

### Screenshots

If applicable, add screenshots to help explain your problem.

### Host / Environment

_No response_

### Additional context

_No response_

### Relevant log output

_No response_

--- a/manifests/2.1.0/opensearch-2.1.0.yml
+++ b/manifests/2.1.0/opensearch-2.1.0.yml
@@ -65,7 +65,7 @@ components:
     ref: '2.1'
     checks:
       - gradle:properties:version
-      - gradle:dependencies:opensearch.version: plugin
+      - gradle:dependencies:opensearch.version: opensearch-sql-plugin
   - name: opensearch-observability
     repository: https://github.com/opensearch-project/observability
     ref: '2.1'

